common/src/androidTest/java/com/microsoft/identity/common/internal/platform/AndroidDevicePoPManagerTests.java

```diff
@@ -474,7 +475,7 @@ public void testHasCertificateChain24() throws ClientException {
     }
 
     @Test
-    @RequiresApi(Build.VERSION_CODES.N)
+    @SdkSuppress(minSdkVersion = Build.VERSION_CODES.N)
```

Comment:
what's the difference between these annotations?

Reply:
<img width="964" height="597" alt="image" src="https://github.com/user-attachments/assets/fa98d2da-8b7e-442c-aa42-1d23faebb8b6" />

Reply:
(AI is a better writer than me... so i'm pasting its response here :P)

Reply:
although... this change was made before i bumped minsdk to 24. we probably don't need any check for < N anymore.